packages/twenty-front/src/modules/workflow/search-variables/utils/parseEditorContent.ts

```diff
@@ -0,0 +1,25 @@
+import { JSONContent } from '@tiptap/react';
+import { isDefined } from 'twenty-ui';
+
+export const parseEditorContent = (json: JSONContent): string => {
+  const parseNode = (node: JSONContent): string => {
+    if (
+      (node.type === 'paragraph' || node.type === 'doc') &&
+      isDefined(node.content)
+    ) {
+      return node.content.map(parseNode).join('');
+    }
+
+    if (node.type === 'text') {
+      return node.text || '';
+    }
+
+    if (node.type === 'variableTag') {
+      return node.attrs?.variable || '';
```

Comment:
If you end up implementing the modification I mentioned about using `data-variable` instead, you should update the code here, too.

Reply:
Nope, as you suggested in your setup, adding the parseHTML in the config will look for the data-variable attribute and set the node attribute variable with the value. Nothing to do there :)